src/patterns/check-a-service-is-suitable/result/index.njk

```diff
@@ -0,0 +1,44 @@
+---
+title: Result – Check a service is suitable
+layout: layout-example-full-page.njk
+---
+
+{% extends "example-wrappers/full-page.njk" %}
+
+{% from "govuk/components/back-link/macro.njk" import govukBackLink %}
+{% from "govuk/components/service-navigation/macro.njk" import govukServiceNavigation %}
+{% from "govuk/components/phase-banner/macro.njk" import govukPhaseBanner %}
+{% from "govuk/components/button/macro.njk" import govukButton %}
+
+{% block header %}
```

Comment:
The `header` block isn't available here to override, hence the fact this service navigation isn't displaying in the example.

I don't think we need to show service navigations - our [minimum required GOV.UK page](https://deploy-preview-4784--govuk-design-system-preview.netlify.app/styles/page-template/#default) does not include service nav, after all.

However, there is the wider issue that to recreate some of these images as code, we might need to tweak the header or footer in some cases, requiring a different approach.

Reply:
Let's leave Service nav out for now but I think it would be useful to have more 'This is what service nav looks like on a site without navigation' examples in place before we remove the options in the header.